### PR TITLE
Development

### PR DESCRIPTION
**Pull request by shivam-tripathi**
Add basic development structure.

### CODE REVIEW

File: .babelrc [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    "transform-object-rest-spread",
+    "transform-es2015-modules-commonjs",
+    "transform-class-properties",
+    "array-includes"
+  ],
+  "presets": [
+    [
+      "env", {
+        "targets": {
+          "node": "8.0"
```
Comment (LordSputnik) [inline .babelrc]: Why not `"node": "current"`?

https://babeljs.io/docs/plugins/preset-env/

File: package.json [hunk 2]
```diff
@@ -0,0 +1,47 @@
+{
+	"name": "bookbrainz-import",
+	"version": "1.0.0",
+	"description": "Import scripts for various data sources ",
```
Comment (LordSputnik) [inline package.json]: Extra space at end of this line

File: package.json [hunk 3]
```diff
@@ -0,0 +1,47 @@
+{
+	"name": "bookbrainz-import",
+	"version": "1.0.0",
```
Comment (LordSputnik) [inline package.json]: Maybe 0.1.0 - save 1.0.0 for the final release during GSOC.

File: package.json [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+	"description": "Import scripts for various data sources ",
+	"main": "index.js",
+	"scripts": {
+		"build": "rimraf lib/* && babel src --out-dir lib",
+		"nodemon": "nodemon --watch './src/' --exec 'npm run build'",
+		"lint": "eslint ./src"
+	},
+	"repository": {
+		"type": "git",
+		"url": "yes "
```
Comment (LordSputnik) [inline package.json]: Does this need an actual URL?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,47 @@
...
+	},
+	"keywords": [
+		"bookbrainz-import"
```
Comment (LordSputnik) [inline package.json]: `bookbrainz`, `book`, `data`, `import` might be more useful keywords here.

File: package.json [hunk 6]
```diff
@@ -0,0 +1,47 @@
...
+	],
+	"author": "Shivam Tripathi",
+	"license": "ISC",
```
Comment (LordSputnik) [inline package.json]: Needs updating to `GPL-2.0+`

File: .babelrc (hunk 1 above)
Comment (shivam-tripathi) [reply .babelrc]: @LordSputnik I used the setting for this same as in bookbrainz-data-js. I'll change it.

File: package.json (hunk 4 above)
Comment (shivam-tripathi) [reply package.json]: Ah, yes. This is the default value. I'll change it.

**Pull request merged**